src/main/java/org/springframework/data/mapping/model/SimpleTypeHolder.java

```diff
@@ -155,7 +155,7 @@ public boolean isSimpleType(Class<?> type) {
 			return isSimpleType;
 		}
 
-		if (type.getName().startsWith("java.lang")) {
+		if (type.getName().startsWith("java.lang") | type.getName().startsWith("java.time")) {
```

Comment:
No double pipe for a reason?

Reply:
Yikes! Consider this one a typo. That's fixed now.